.code-samples.meilisearch.yaml

```diff
@@ -322,26 +335,50 @@ search_parameter_guide_retrieve_1: |-
   client.index("movies").search(searchRequest);
 search_parameter_guide_crop_1: |-
   SearchRequest searchRequest =
-    new SearchRequest("shifu")
-      .setAttributesToCrop(new String[] {"overview"})
-      .setCropLength(5);
+          SearchRequest.builder()
+                  .q("shifu")
+                  .attributesToCrop(new String[] {"overview"})
+                  .cropLength(5)
+                  .build();
   client.index("movies").search(searchRequest);
 search_parameter_guide_crop_marker_1: |-
   SearchRequest searchRequest =
-    new SearchRequest("shifu")
-      .setAttributesToCrop(new String[] {"overview"})
-      .setCropMarker("[…]");
+          SearchRequest.builder()
+                  .q("shifu")
+                  .attributesToCrop(new String[] {"overview"})
+                  .cropMarker("[…]")
+                  .build();
   client.index("movies").search(searchRequest);
 search_parameter_guide_highlight_1: |-
   SearchRequest searchRequest =
     new SearchRequest("winter feast").setAttributesToHighlight(new String[] {"overview"});
   client.index("movies").search(searchRequest);
 search_parameter_guide_highlight_tag_1: |-
   SearchRequest searchRequest =
-    new SearchRequest("winter feast")
-    .setAttributesToHighlight(new String[] {"overview"})
-    .setHighlightPreTag("<span class=\"highlight\">")
-    .setHighlightPostTag("</span>");
+          SearchRequest.builder()
+                  .q("winter feast")
+                  .attributesToHighlight(new String[] {"overview"})
+                  .highlightPreTag("<span class=\"highlight\">")
+                  .highlightPostTag("</span>")
+                  .build();
+  client.index("movies").search(searchRequest);
+search_parameter_guide_matches_1: |-
```

Comment:
I did not find this code sample, so I think we can remove it!

Reply:
It's from this issue I think it could be alright to put it in the same time
https://github.com/meilisearch/meilisearch-java/issues/460